### PR TITLE
Mostly Go style changes.

### PR DESCRIPTION
**Pull request by bendory**

### CODE REVIEW

File: builder/builder/gce.go [hunk 1]
```diff
@@ -134,23 +135,10 @@ func (s *Server) NewGCEService(ctx context.Context) error {
...
-	return fmt.Sprintf("%s-%s", instanceNamePrefix, b)
-}
-
-// NewInstance starts a Windows VM on GCE and returns host, username, password.
-func (s *Server) NewInstance() error {
+// newInstance starts a Windows VM on GCE and returns host, username, password.
+func (s *Server) newInstance() error {
 	scmd := startupCmd // TODO: find better way to take address of const
-	name := RandomName()
+	name := uuid.New()
```
Comment (nof20) [inline builder/builder/gce.go]: Understand the need to replace hand-rolled code with standard UUID implementation, but if someone (not necessarily the same user) happens to glance at the GCE dashboard there's no clue about what the host is doing.  How about `fmt.Sprintf("windows-builder-%s", uuid.New())` or similar?

**Review by nof20 — COMMENTED** [review]
Thanks so much for this!  I'm on board with everything except the VM name, what do you think?

**Comment by bendory** [comment]
updated VM name

**Review by nof20 — APPROVED** [review]
Cheers!

**Pull request merged**